docs/content/configuration/csi_ug_config_replication_find_systemid.md

```diff
@@ -0,0 +1,8 @@
+# Finding a `system_id`
+
+Find the storage system `system_id` parameter on your storage system in order to create a VolumeReplicationClass YAML file, enabling replication.
+
+For finding the `system_id` parameter on your Spectrum Virtualize storage system, use the `lspartnership` command.
+
+For more information, see **Command-line interface** > **Copy Service commands** > **lspartnership** within your product documentation on [IBM Docs](https://www.ibm.com/docs/en).
```

Comment:
```suggestion
For more information, see  **Copy Service commands** > **Command-line interface** > **lspartnership** within your product documentation on [IBM Docs](https://www.ibm.com/docs/en).
```

does the first `>` mean "search"?

Reply:
It's how how we show the menuscape / breadcrumbs for the pages.